cmd/agent/subcommands/status/command.go

```diff
@@ -138,18 +129,7 @@ func setIpcURL(cliParams *cliParams) url.Values {
 }
 
 func renderResponse(res []byte, cliParams *cliParams) error {
-	var s string
-
-	// The rendering is done in the client so that the agent has less work to do
-	if cliParams.prettyPrintJSON {
-		var prettyJSON bytes.Buffer
-		json.Indent(&prettyJSON, res, "", "  ") //nolint:errcheck
-		s = prettyJSON.String()
-	} else if cliParams.jsonStatus {
-		s = string(res)
```

Comment:
Previously we where not scrubbing the JSON. We are now scrubbing it, I think is a breaking change. What are you thoughts on that @louis-cqrl?